### PR TITLE
updates intervals, adds missing params and test yaml

### PR DESCRIPTION
**Pull request by chrisamiller**
Updates interval list params to more closely resemble those in the other workflows.  Also exposes a few missing params, and adds a test yaml

### CODE REVIEW

**Comment by chrisamiller** [comment]
closes https://github.com/genome/analysis-workflows/issues/928

File: example_data/tumor_only_wgs.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+- set
+variants_to_table_genotype_fields:
+- GT
+- AD
+- AF
+- DP
+
+
+
+
```
Comment (tmooney) [inline example_data/tumor_only_wgs.yaml:102]: There's some inconsistent whitespace in this file.  Some items are separated by one blank line, some by two, some by none, and four appear here at the end.

**Review by tmooney — COMMENTED** [review]
Was this PR meant to also include some changes to the CWL itself?  Some of the params in this example YAML don't match what's in `tumor_only_wgs.cwl`.

**Comment by chrisamiller** [comment]
ugh - thank you, the cwl didn't get pushed.  Updated the whitespace as well

**Comment by gschang** [comment]
+1

**Review by tmooney — APPROVED** [review]
+1, if I were to say anything I might say adding doc lines to some of the inputs makes me wonder if we should add some to the other new inputs being added here!

**Pull request merged**